### PR TITLE
Team service

### PR DESCRIPTION
**Pull request by gbougeard**

### CODE REVIEW

File: src/main/scala/tv/teads/github/api/services/MemberService.scala [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+        logger.error(s"Fetching organization $org members failed, status code: ${statusCode.intValue}")
+        Nil
+    }
+  }
+
+  def fetchDefaultOrgMembers(implicit refFactory: ActorRefFactory, ec: ExecutionContext): Future[List[Member]] = {
+    fetchOrgMembers(configuration.organization)
+  }
+
+//  def isMemberOf(org: String, username:String)(implicit refFactory: ActorRefFactory, ec: ExecutionContext): Future[Boolean] = {
```
Comment (pdalpra) [inline src/main/scala/tv/teads/github/api/services/MemberService.scala]: commented code

File: src/test/scala/tv/teads/github/api/services/MemberServiceSpec.scala [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    }
+  }
+  it should "be able to fetch default (configured) organization members" in {
+
+    whenReady(MemberService.fetchDefaultOrgMembers) { org ⇒
+      org should not be empty
+    }
+  }
+
+  //  it should "be able to fetch user BobTheBot-teads' organization" in {
```
Comment (pdalpra) [inline src/test/scala/tv/teads/github/api/services/MemberServiceSpec.scala]: commented code

File: src/test/scala/tv/teads/github/api/services/TeamServiceSpec.scala [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+import scala.concurrent.duration._
+
+class TeamServiceSpec extends BaseSpec {
+
+  override implicit val patienceConfig = PatienceConfig(30 seconds, 1 second)
+
+  "Team Service" should "be able to fetch an organization teams" in {
+
+    whenReady(TeamService.fetchOrgTeams("ebuzzing")) { list ⇒
+      //      println(list)
```
Comment (pdalpra) [inline src/test/scala/tv/teads/github/api/services/TeamServiceSpec.scala]: commented code

File: src/test/scala/tv/teads/github/api/services/TeamServiceSpec.scala [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+      list should not be empty
+    }
+  }
+  it should "be able to fetch default organization teams" in {
+
+    whenReady(TeamService.fetchDefaultOrgTeams) { list ⇒
+      //      println(list)
```
Comment (pdalpra) [inline src/test/scala/tv/teads/github/api/services/TeamServiceSpec.scala]: commented code

File: src/test/scala/tv/teads/github/api/services/TeamServiceSpec.scala [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+      list should not be empty
+    }
+  }
+  it should "be able to fetch teams member" in {
+
+    whenReady(TeamService.fetchTeamMembers(1276819)) { list ⇒
+      //      println(list)
```
Comment (pdalpra) [inline src/test/scala/tv/teads/github/api/services/TeamServiceSpec.scala]: commented code

File: src/test/scala/tv/teads/github/api/services/MemberServiceSpec.scala (hunk 2 above)
Comment (gbougeard) [inline src/test/scala/tv/teads/github/api/services/MemberServiceSpec.scala]: this one is from another PR (yes I did dependants PRs..)

Comment (pdalpra) [inline src/test/scala/tv/teads/github/api/services/MemberServiceSpec.scala]: commented code is still there :)

**Comment by gbougeard** [comment]
LGTY? need to merge #18 first

**Comment by pdalpra** [comment]
you'll need to get the Member Service commit out :)

**Comment by pdalpra** [comment]
LGTM

**Pull request merged**